### PR TITLE
logging - add simple upgrade job

### PR DESCRIPTION
**Pull request by richm**

### CODE REVIEW

**Comment by richm** [comment]
/ci/rehearse/openshift/origin-aggregated-logging/master/upgrade-full-integ-aws

**Comment by richm** [comment]
multi-tenancy flake
/retest

**Comment by richm** [comment]
flake
/retest

**Comment by richm** [comment]
This looks like the flake problem that should be fixed by https://github.com/openshift/cluster-logging-operator/pull/230
/retest

File: ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml [hunk 1]
```diff
@@ -221,3 +221,71 @@ presubmits:
...
+        name: ""
+        resources:
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/upgrade-full-integ-aws-cluster-profile
+          name: cluster-profile
+        - mountPath: /usr/local/upgrade-full-integ-aws
+          name: job-definition
+          subPath: cluster-launch-installer-src.yaml
```
Comment (ewolinetz) [inline ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml]: just out of curiosity, do we have any other jobs that use this job template?

**Comment by ewolinetz** [comment]
overall LGTM, just have a quick question

File: ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml (hunk 1 above)
Comment (richm) [reply ci-operator/jobs/openshift/origin-aggregated-logging/openshift-origin-aggregated-logging-master-presubmits.yaml]: yes - the `full-integ-aws` test uses this template

**Pull request merged**